app/parse.py

```diff
@@ -8,8 +21,46 @@ class Quote:
     tags: list[str]
 
 
+def parse_single_quote(quote_soup: BeautifulSoup) -> Quote:
+    return Quote(
+        text=quote_soup.select_one(".text").text,
+        author=quote_soup.select_one(".author").text,
+        tags=[tag.text for tag in quote_soup.select("a.tag")]
+    )
+
+
+def parse_single_page(page_quotes: BeautifulSoup) -> [Quote]:
```

Comment:
You return a `list` of `Quote` instances as well. Pay greater attention to the return type annotations in the future.